app/assets/sass/components/_cookie-banner-new.scss

```diff
@@ -0,0 +1,31 @@
+.app-cookie-banner-new {
+  @include govuk-font(16);
+  @include govuk-text-colour;
+
+  box-sizing: border-box;
+  width: 100%;
+
+  padding-top: govuk-spacing(3);
+  padding-right: govuk-spacing(3);
+  padding-bottom: govuk-spacing(3);
+  padding-left: govuk-spacing(3);
+}
+
+.app-cookie-banner-new__message {
+  margin: 0;
+  @include govuk-width-container;
+}
+
+.app-cookie-banner-new__button {
+  width: 100%;
+
+  @media (min-width: 48.0625em) {

```

Comment:
You should be able to use

```
@include govuk-media-query($from: tablet) { }
```

See example here https://github.com/hmrc/hmrc-frontend/blob/df356955199a820a6ec4cb761956b8f275e6da16/src/components/status-tags-in-task-list-pages/_status-tags-in-task-list-pages.scss#L13